ops/helmfiles/config/frontend.yaml

```diff
@@ -27,64 +27,37 @@ collator:
     tag: 456a53b8bdd546d99507884cf4a82b844366c2eb-fast
   disableChainspecGeneration: true
   customChainspecUrl: https://raw.githubusercontent.com/gasp-xyz/chainspec/eb43d6a85f5152cc5f4bae0996bf7b74ad1c516d/rollup-dev-env-chainspecs/rollup-fungible-456a53b8-raw.json
-  storageClass: standard
-  storageSize: 5Gi
+  storageClass: node-data-volume-retain
+  storageSize: 50Gi
   enableServiceMonitorCollectors: true
   wsHealthExporter:
-    enabled: false
+    enabled: true
+    image: paritytech/ws-health-exporter:bc071a44-20230830
+    env:
+      WSHE_NODE_MIN_PEERS: 2
+  tolerations:
+    - key: production
+      operator: Equal
+      value: "true"
+      effect: NoSchedule
   collator01:
-    # Identity ID: 12D3KooWAfm5cK7DuUa2zh9SrSoHr2fsXXYbLALWXs5ob9mTDKHJ
-    customNodeKey: "5ad94653edf592f59d1b54960cda1334596cc3dcab8dd96adb334d3e22e912ef"
-    bootnodeAddr: /dns/collator-02-p2p/tcp/30333/p2p/12D3KooWFz4zM26EKYTpX98GhLhS3KLv7mUyvpYXCdzxdrorSVCu
+    # Identity ID: 12D3KooWJCggfzsr6LAdXSdfgFLxsMtgCq9qSq79vqYtdG2oT8Za
+    customNodeKey: "a9a20a5fa298fdd68ae5516c60c060c419e37bae4440d6f52ae3439162522fa2"
+    bootnodeAddr: /dns/collator-02-p2p/tcp/30333/p2p/12D3KooWB7YXuwmv55wy5UFDe9yRGYJFPk1r3DpFMSyFx5KqMi3G
     extraArgs:
       - --alith
       - --pruning=archive
+    nodeSelector:
+      production: "true"
   collator02:
-    # Identity ID: 12D3KooWFz4zM26EKYTpX98GhLhS3KLv7mUyvpYXCdzxdrorSVCu
-    customNodeKey: "bcaf1bb787b48156aadaf318603e5b37e637883edafb850912e364e913e2c35b"
-    bootnodeAddr: /dns/collator-01-p2p/tcp/30333/p2p/12D3KooWAfm5cK7DuUa2zh9SrSoHr2fsXXYbLALWXs5ob9mTDKHJ
+    # Identity ID: 12D3KooWB7YXuwmv55wy5UFDe9yRGYJFPk1r3DpFMSyFx5KqMi3G
+    customNodeKey: "bf08651f3aa5b9fed7ce2e82a5fce8c31f1889e93e3019476b3052f33d57116f"
+    bootnodeAddr: /dns/collator-01-p2p/tcp/30333/p2p/12D3KooWJCggfzsr6LAdXSdfgFLxsMtgCq9qSq79vqYtdG2oT8Za
     extraArgs:
       - --baltathar
       - --pruning=archive
-
```

Comment:
so we need this in fungible and and do not it in frontend? I though they are almost the same?